### ISSUE TITLE
Why not use the official implementation of group normalization in pytorch?

### ISSUE THREAD

**Issue by Wenlong-Shi** [wolny/pytorch-3dunet#12]
What's the difference between unet3d.groupnorm and torch.nn.GroupNorm?

**Comment by wolny** [comment]
Hey @Wenlong-Shi,

there is no real different, thanks for pointing that out. I think that the official pytorch implementation wasn't available at the time when I open sourced the model so I hacked something myself.
I've removed my implementation and use the official one now. Thanks!
